tests/sqlalchemy_test_suite/test_suite.py

```diff
@@ -0,0 +1,499 @@
+        """
+        pass
+
+    @pytest.mark.skip("need investigation")
+    def test_fk_column_order(self):
+        """
+        >       assert a == b, msg or "%r != %r" % (a, b)
+        E       AssertionError: ['attr', 'id', 'name'] != ['name', 'id', 'attr']
+        """
+        pass
+
+
+class CTETest(_CTETest):
+    @pytest.mark.skip("need investigation")
+    def test_delete_from_round_trip(self, connection):
+        """
+        E       sqlalchemy.exc.ProgrammingError: (snowflake.connector.errors.ProgrammingError) 001003 (42000): SQL compilation error:
+        E       syntax error line 5 at position 1 unexpected 'DELETE'.
+        E       [SQL: WITH some_cte AS
+        E       (SELECT some_table.id AS id, some_table.data AS data, some_table.parent_id AS parent_id
+        E       FROM some_table
+        E       WHERE some_table.data IN (%(data_1_1)s, %(data_1_2)s, %(data_1_3)s))
+        E        DELETE FROM some_other_table USING some_cte WHERE some_other_table.data = some_cte.data]
+        E       [parameters: {'data_1_1': 'd2', 'data_1_2': 'd3', 'data_1_3': 'd4'}]
+        E       (Background on this error at: https://sqlalche.me/e/14/f405)
+        """
+        pass
+
+    @pytest.mark.skip("need investigation")
+    def test_delete_scalar_subq_round_trip(self, connection):
+        """
+                E       sqlalchemy.exc.ProgrammingError: (snowflake.connector.errors.ProgrammingError) 001003 (42000): SQL compilation error:
+        E       syntax error line 5 at position 1 unexpected 'DELETE'.
+        E       [SQL: WITH some_cte AS
+        E       (SELECT some_table.id AS id, some_table.data AS data, some_table.parent_id AS parent_id
+        E       FROM some_table
+        E       WHERE some_table.data IN (%(data_1_1)s, %(data_1_2)s, %(data_1_3)s))
+        E        DELETE FROM some_other_table WHERE some_other_table.data = (SELECT some_cte.data
+        E       FROM some_cte
+        E       WHERE some_cte.id = some_other_table.id)]
+        E       [parameters: {'data_1_1': 'd2', 'data_1_2': 'd3', 'data_1_3': 'd4'}]
+        E       (Background on this error at: https://sqlalche.me/e/14/f405)
+        """
+        pass
+
+
+class DateTest(_DateTest):
+    @pytest.mark.skip("need investigation")
+    def test_select_direct(self, connection):
+        """
+        >       assert a == b, msg or "%r != %r" % (a, b)
+        E       AssertionError: '2012-10-15' != datetime.date(2012, 10, 15)
+        """
+        pass
+
+
+class DateTimeCoercedToDateTimeTest(_DateTimeCoercedToDateTimeTest):
+    @pytest.mark.skip("need investigation")
+    def test_select_direct(self, connection):
+        """
+        >       assert a == b, msg or "%r != %r" % (a, b)
+        E       AssertionError: '2012-10-15 12:57:18' != datetime.datetime(2012, 10, 15, 12, 57, 18)
+        """
+        pass
+
+
+class DateTimeMicrosecondsTest(_DateTimeMicrosecondsTest):
+    @pytest.mark.skip("need investigation")
```

Comment:
Are you not going to investigate these in this PR? If not then please file a follow-up task!